frontend/device/pocketbook/device.lua

```diff
@@ -110,11 +110,9 @@ function PocketBook:init()
             if fb:isB288() then
                 logger.dbg("mxcfb: Disabling hwinvert on B288 chipset")
                 self.canHWInvert = no
-                -- GL16 glitches with hwrot
-                if fb.wf_level == 3 then
```

Comment:
Nothing device-specific? (I mean of the `if something ~= known_working then` variety.)

Reply:
Probably, but hard to tell, and it's caused no end of weird issues on some devices/FW combos for a while anyway :/.

Someone who actually cares about PB could probably make that user-configurable instead.